msvc/README.md

```diff
@@ -69,81 +71,34 @@ You can pass in default arguments to the csound instance by right-clicking on "c
-|BUILD_INSTALLER                    |                   |          |
-|BUILD_JACK_OPCODES                 |                   | Disabled |
-|BUILD_JAVA_INTERFACE               |                   | Working |
-|BUILD_LINEAR_ALGEBRA_OPCODES       | Eigen             | Enabled/Not being found |
-|BUILD_LUA_INTERFACE                |                   |          |
-|BUILD_LUA_OPCODES                  |                   |          |
-|BUILD_MULTI_CORE                   |                   |          |
-|BUILD_OSC_OPCODES                  | Liblo             | Enabled/Needs work |
-|BUILD_P5GLOVE_OPCODES              |                   |          |
-|BUILD_PYTHON_INTERFACE             |                   |          |
-|BUILD_PYTHON_OPCODES               |                   |          |
-|BUILD_RELEASE                      |                   | Working |
-|BUILD_SERIAL_OPCODES               |                   |          |
-|BUILD_STATIC_LIBRARY               |                   | Working |
-|BUILD_STK_OPCODES                  |                   |          |
-|BUILD_TESTS                        |                   | Enabled/Needs work |
-|BUILD_UTILITIES                    |                   | Working |
-|BUILD_VIRTUAL_KEYBOARD             | FLTK              | Enabled/Needs work |
-|BUILD_WIIMOTE_OPCODES              |                   |          |
-|BUILD_WINDSOUND                    |                   |          |
-|USE_ALSA                           |                   | Disabled |
-|USE_ATOMIC_BUILTIN                 |                   | Enable/Not working |
-|USE_AUDIOUNIT                      |                   | Disabled |
-|USE_COMPILER_OPTIMIZATIONS         |                   | Working |
-|USE_COREMIDI                       |                   | Disabled |
-|USE_DOUBLE                         | -                 | Working |
-|USE_FLTK                           |                   | Enabled/Not being found |
-|USE_GETTEXT                        | VCPKG             | Enabled/Not being found |
-|USE_IPMIDI                         |                   |          |
-|USE_JACK                           |                   | Disabled |
-|USE_LIB64                          |                   | Working |
-|USE_LRINT                          |                   | Disabled |
-|USE_OPEN_MP                        |                   |          |
-|USE_PORTAUDIO                      |                   | Working |
-|USE_PORTMIDI                       | Manual build      | Enabled/Needs work |
-|USE_PULSEAUDIO		                  |                   | Disabled |
-|USE_SYSTEM_PORTSMF                 |                   |          |
-
 ## Work in progress / work to do
-1. Python integration into build. Most likely user will need to install it via msi
-2. PortMIDI/SMF manual build script needs to be created
+1. [Fixed] Python integration into build. Most likely user will need to install it via msi
```

Comment:
Yeah agreed. I was mainly keeping it there for historic reasons but probably not point. I'll remove it